.tool-versions

```diff
@@ -0,0 +1 @@
+terraform 1.5.1
```

Comment:
We are not using asdf on this project yet. I think it's more worth to propose it on a different issue to open a discussion, and then see what happens. So to merge this code, please, delete this file 🙏🏼